src/algebra/star/pi.lean

```diff
@@ -63,6 +63,16 @@ by { ext x, cases x; simp }
 
 end function
 
+/- Extra instance to short-circuit type class resolution.
+For unknown reasons, this is necessary for certain inference problems. E.g., for this to succeed:
+```lean
+example (β X : Type*) [normed_add_comm_group β] [normed_space ℝ β] : module ℝ (X → β) :=
+infer_instance
```

Comment:
This example is unrelated! To come up with a real example, come up with soemthing that fails if you add `attribute [-instance] function.star_module`

Reply:
Oops! Fixed now.